content/lessons/chapter-3/coop-1.tsx

```diff
@@ -0,0 +1,125 @@
+'use client'
+
+import { useTranslations, useSaveAndProceed } from 'hooks'
+import { Title } from 'ui'
+import { Button } from 'shared'
+import Avatar from 'components/Avatar'
+import { useMediaQuery } from 'hooks'
+import { useState, useEffect } from 'react'
+
+export const metadata = {
+  title: 'chapter_three.coop_vs_bitrey_loose.title',
+  image: '/assets/images/chapter-2-intro-1.jpg',
+  key: 'CH3COO1',
+}
+
+export default function Coop1({ lang }) {
+  const t = useTranslations(lang)
+
+  const saveAndProceed = useSaveAndProceed()
+  const isSmallScreen = useMediaQuery({ width: 767 })
+  const [playersFound, setPlayersFound] = useState(false)
+  const [players, setPlayers] = useState([
+    {
+      username: 'You',
+      avatar: '/assets/avatars/1.png',
```

Comment:
We should probably try to take the avatar of the currently logged in user (if it exists) and fallback to this one.